bot/exts/evergreen/stackoverflow.py

```diff
@@ -0,0 +1,45 @@
+from html import unescape
+from urllib.parse import quote_plus
+
+from discord import Embed
+from discord.ext.commands import Bot, Cog, Context, command, cooldown
```

Comment:
The prevailing style for Sir Lancebot is to do `from discord.ext import commands` and then use `commands.Bot` as an example, rather than importing the individual pieces like this.